src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -18,6 +18,7 @@ public interface IPackageMetadata
         IEnumerable<string> Owners { get; }
         Uri IconUrl { get; }
         Uri LicenseUrl { get; }
+        string Icon { get; }
```

Comment:
what's the reasoning behind this.
It's not sorted alphabetically nor is it close to the other prop with equivalent meaning.

Reply:
I moved the Icon property there because I had a merge conflict when I was rebasing. The reason for putting the Icon property near there is because it is _near_ the properties with simple, system types (int, string). I will move it back to minimize changeset.